pkg/connector/tickets.go

```diff
@@ -21,6 +22,168 @@ import (
 
 const maxProjects = 100
 
+// example https://developer.atlassian.com/server/jira/platform/jira-rest-api-example-create-issue-7897248/
+func (d *Connector) customFieldSchemaToMetaField(field *v2.TicketCustomField) (interface{}, error) {
+	if field == nil {
+		return nil, nil
+	}
+
+	type JiraPickerStruct struct {
+		Id string `json:"id"`
+	}
+
+	pickObjects := []*JiraPickerStruct{}
+
+	switch v := field.GetValue().(type) {
+	case *v2.TicketCustomField_StringValue:
+		strValue := v.StringValue.GetValue()
+		if d.cfIdToJiraType[field.GetId()] == jira.TypeUser {
```

Comment:
is this always safe?, should it be `if v, ok := d.cfIdToJiraType[field.GetId()]; ok {...`?

also below

Reply:
https://github.com/ConductorOne/baton-jira-datacenter/pull/28/commits/901e4d7306cf01130eec0d06a15154b51aaec709